spec/controllers/credentials_controller_spec.rb

```diff
@@ -92,17 +82,7 @@
           let(:new_password) { +"the\npassword" }
           let(:errors) {
             {
-              error: {
-                code: "validation_failed",
-                message: "password cannot contain a newline",
-                details: [
-                  {
-                    code: "validation_failed",
-                    target: "password",
-                    message: "cannot contain a newline"
-                  }
-                ]
-              }
+              Reason: Errors::Conjur::InsufficientPasswordComplexity.new.to_s
```

Comment:
You have two tests that are now redundent - empty password and one containing new line, since we consolidated those errors to one InsufficientPasswordComplexity, yielding same error message that don't need to be seperated.
However! there is something we need to check - why is new line is not escaped? are there any other issues for example: `<script>123!@#Test` and are we now enabline \n in password? I think that `Test\nWillWork123' and thus allowing passwords that we didn't allow before, maybe breaking UI also.